src/protocolsupport/protocol/typeremapper/tile/TileEntitySkullRemapper.java

```diff
@@ -3,19 +3,25 @@
 import java.util.List;
 import java.util.function.Consumer;
 
+import org.apache.logging.log4j.core.net.Protocol;
```

Comment:
?????

Reply:
I have no idea where that one came from, maybe because I was figuring out if ProtocolAPI could detect PE before I chose the method I did now and it auto imported something else?